### PR TITLE
it is tha a file belong to IS_A_Relation_Hierarchical

### PR DESCRIPTION
**Pull request by varaprasadram1**
Hi @yvpram, I am adding a IS_A_Relation_Hierarchical file please verify my code 
regard,
@varaprasadram1

### CODE REVIEW

File: src/main/java/oops/IS_A_Relation_Hierarchical.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	public void p2assets() {
+		int car=2;
+		System.out.println("car "+car);
+	}
+
+	public void c1assets() {
+		int c1car=2;
+		System.out.println("c1car "+c1car);
+	}
+
```
Comment (yvpram) [inline src/main/java/oops/IS_A_Relation_Hierarchical.java]: Hi @varaprasadram1,  remove this empty space line for code looks good.
 regards,
 @yvpram.

**Review by yvpram — COMMENTED** [review]
Hi @varaprasadram1, remove this empty space line for code looks good.
regards,
@yvpram.

File: src/main/java/oops/IS_A_Relation_Hierarchical.java (hunk 1 above)
Comment (varaprasadram1) [reply src/main/java/oops/IS_A_Relation_Hierarchical.java]: okay change now

**Review by yvpram — APPROVED** [review]
I approve this, @varaprasadram1

**Pull request merged**